src/layouts/Base.astro

```diff
@@ -80,37 +89,28 @@ const pathname = Astro.url.pathname.replace(/\/$/, "")
 				</div>
 			</BackgroundGridContainer>
 
-			<main
-				class="container flex flex-col pt-8 sm:px-6 sm:pt-12 lg:px-8 lg:pt-16"
-			>
-				<ul class="templates mb-24 flex flex-col gap-24">
+			<main class="container">
+				<nav class="flex flex-wrap border-b border-astro-gray-500">
+					{
+						[...Astro.props.examples.entries()].map(([category]) => (
+							<a
+								href={posix.join(Astro.props.basePath, category)}
```

Comment:
Looks like these links need a `/` prepended to avoid relative path bugs, going to `/integrations` and then clicking another tab breaks without it

I end up copy/pasting these helpers into most projects eventually to handle joining + prepending `/`

```ts
import { posix } from "node:path"

export function prependSlash(path: string) {
	return path.startsWith("/") ? path : `/${path}`
}

export function joinPath(...parts: string[]) {
	return prependSlash(posix.join(...parts))
}
```